src/main/java/Message.java

```diff
@@ -0,0 +1,25 @@
+public class Message {
+    public static void printingGreeting() {
```

Comment:
Consider naming the function's name as `printGreeting()` instead of `printingGreeting()`.